src/Views/Homepage.vala

```diff
@@ -153,6 +153,58 @@ namespace AppCenter {
                 new Thread<void*> ("update-recent-carousel", run);
             });
 
+            var trending_label = new Gtk.Label (_("Trending"));
+            trending_label.get_style_context ().add_class ("h4");
+            trending_label.xalign = 0;
+            trending_label.margin_start = 10;
+
+            var trending_carousel = new Widgets.Carousel ();
+
+            var trending_grid = new Gtk.Grid ();
+            trending_grid.margin = 2;
+            trending_grid.margin_top = 12;
+            trending_grid.attach (trending_label, 0, 0, 1, 1);
+            trending_grid.attach (trending_carousel, 0, 1, 1, 1);
+
+            var trending_revealer = new Gtk.Revealer ();
+            trending_revealer.add (trending_grid );
+
+            houston.get_trending.begin ((obj, res) => {
+                var updated_ids = houston.get_updated.end (res);
+                List<AppCenterCore.Package> packages_for_carousel = new List<AppCenterCore.Package> ();
```

Comment:
GLib list or GeeList? (Should use Gee if possible, and you can use var at the beginning